frontend/src/pages/Login.js

```diff
@@ -1,12 +1,12 @@
 import { Link, useNavigate } from "react-router-dom";
-import LoginForm from "../components/loginForm";
+import LoginForm from "../components/LoginForm";
```

Comment:
- you change the import to pascal case but you did not the rename the actual file e.g. **LoginForm.js** that results the following error below (apply to related)
![image](https://user-images.githubusercontent.com/73863685/188393200-42ef7dbd-2d48-45cf-9ab6-d191bee869c4.png)
![image](https://user-images.githubusercontent.com/73863685/188393244-a614081b-d824-4037-8126-c6186c626ee7.png)